general/platforms.rst

```diff
@@ -233,6 +233,11 @@ please refer to the file MAINTAINERS_ for contact details for various platforms.
      - Yes
      - Yes
 
+   * - `Rockchip RK3588 <https://rock-chips.com/a/en/products/RK35_Series/2022/0926/1660.html>`_
```

Comment:
This URL doesn't work, but if you prepend it with www it does work, like this. Please update this and then we're good to merge this.
https://www.rock-chips.com/a/en/products/RK35_Series/2022/0926/1660.html